### PR TITLE
glow animation on view update

### PR DESCRIPTION
**Pull request by hthayko**

### CODE REVIEW

**Comment by wilsonianb** [comment]
Looks cool. What do you think about glowing green for new connections?

**Comment by hthayko** [comment]
that's a great idea. Should we glow green when a number goes up and use red when it goes down ? for all of the 3 animated properties: in_connections, out_connections and uptime ?

**Comment by wilsonianb** [comment]
That sounds good to me

File: src/services/Peers.js [hunk 1]
```diff
@@ -56,5 +58,43 @@ module.exports = {
...
+    });
+    var ret = _.map(newPeers, function(p) {
+      return { new: p, old: oldPeersByPubKey[p.public_key]};
+    });
+    return ret;
+  },
+
+  animateChange: function(propertyArray) {
+    function animate($element, oldValue, newValue) {
+      var originalColor = '#333';
```
Comment (wilsonianb) [inline src/services/Peers.js]: Can you get this from `$element`?

Comment (hthayko) [inline src/services/Peers.js]: I can, but animate function assumes it gets an element and oldValue, newValue to animate between

Comment (wilsonianb) [inline src/services/Peers.js]: I'm just thinking that if we ever switch from #333, we would only have to change the CSS

Comment (hthayko) [inline src/services/Peers.js]: oh, sorry, I thought you were asking about getting oldValue and NewValue from $element. 
I will fix the originalColor right away

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**